File: benchmark/template/programs/cpp/adversarial/ADV-05_C.cpp
#include <cstdio>
#include <cstdint>

int main() {
    printf("ADV-START\n"); fflush(stdout);
    double d = 1e30;
    std::int32_t v = d;
    printf("OBS=V:%d\n", (int)v); fflush(stdout);
    printf("ADV-END\n"); fflush(stdout);
    return 0;
}

File: benchmark/template/programs/rust/adversarial/ADV-05_C.rs
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let f: f64 = 1e30;
    let v: i32 = f as i32;

    writeln!(out, "OBS=V:{}", v).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
